libs/input-mapping/tests/Functional/DownloadTablesWorkspaceSnowflakeTest.php

```diff
@@ -57,7 +57,7 @@ public function testTablesSnowflakeBackend(): void
         // there were 2 jobs, clone and copy, so should have 2 metrics entries
         $metrics = $result->getMetrics()?->getTableMetrics();
         self::assertNotNull($metrics);
-        self::assertCount(2, $metrics);
+        self::assertCount(2, iterator_to_array($metrics));
```

Comment:
phpunit 9.6 háže

```
Passing an argument of type Generator for the $haystack parameter is deprecated. Support for this will be removed in PHPUnit 10.
```